src/index.html

```diff
@@ -2,16 +2,183 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
+    <meta name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
+    <title>HTML form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>HTML Form</h1>
+  <body class="removeDef">
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fields">
+        <legend>Personal information</legend>
+        <div class="inputs">
+          <label>
+          Surename:
+            <input
+              type="text"
+              name="Surename"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
+        <div class="inputs">
+          <label>
+          Name:
+            <input
+              type="text"
+              name="Name"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
+        <div class="inputs">
+          <label>
+          How old are you?
+            <input type="number"
+              name="Age"
+              max="100"
+              min="1"
+              value="12"
+              required
+            >
+          </label>
+        </div>
+        <div class="inputs">
+          <label>
+          Full date of birth:
+            <input
+              type="date"
+              name="dateOfBirth"
+              required
+            >
+          </label>
+        </div>
+        <div>
+          <label>
+          I accept the term of the aggrement
+            <input
+              type="checkbox"
+              name="acceptTermOfAggrement"
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="fields">
+        <legend>Registration</legend>
+        <div class="inputs">
+          <label>
+          E-mail:
+            <input
+              type="email"
+              name="eMail: "
+              placeholder="email@example.com"
+              required
+            >
+          </label>
+        </div>
+        <div>
+          <label>
+          Password:
+            <input
+              type="password"
+              name="Password"
+              required
+              minlength="5"
+              maxlength="32"
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="fields">
+        <legend>An interesting fact about you!</legend>
+        <div class="inputs">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="loveCats"
+              value="yes"
+            >
+                Yes
```

Comment:
and this one is over-indented
```suggestion
            >
            Yes
```